docs-src/tut-7-rpc.scrbl

```diff
@@ -24,12 +24,39 @@ Reach code.
 Below we will compare the @secref["tut-7"] JavaScript @tech{frontend} with the
 equivalent Python code communicating via RPC, section by section.
 
-First, we need to install the library from the command line:
-@cmd{
-  pip install --upgrade reach-rpc-client
+@(hrule)
+
+First we need to set up our project and install the library from the command
+line:
+
+@verbatim{
+$ mkdir rps-python
+$ cd rps-python
+
+$ printf 'venv/\nbuild/\n' > .gitignore
+
+$ curl -o index.rsh "https://raw.githubusercontent.com/reach-sh/reach-lang/master/examples/tut-7-rpc/server/index.rsh"
+
+$ curl -o reach "https://raw.githubusercontent.com/reach-sh/reach-lang/master/reach"
+
+$ chmod +x ./reach
+
+$ git init
+$ git add .gitignore index.rsh reach
+$ git commit -m 'First commit'
+
+$ python3 -m venv ./venv
+$ . ./venv/bin/activate
+$ pip install --upgrade reach-rpc-client
```

Comment:
I think this is the only line you should include and I think that you should have a `@margin-note` about how it's common to use `venv`

Reply:
I can spend some more time pondering this if you feel like it's worth it, but I actually think the blurb:
```bash
[ -d ./venv ] || python3 -m venv ./venv
source ./venv/bin/activate
pip install --upgrade reach-rpc-client
./reach rpc-dev python3 -u ./index.py
```

from 629a8be39b206769a14c79dcf16af93d82091cbd is probably the best way, albeit still pretty noisy.

The problem is that `pip` is itself a package and not universally distributed with Python, so we can either use the built-in `venv` stdlib module (which _is_ universally distributed) to obtain `pip`, or add even more installation instructions that aren't very relevant to the task at hand.

This blurb also has the virtues of being idempotent (for people who aren't sure what they're copy+pasting and might redo these steps later) and the Right Way from a Python programmer's perspective.

Reply:
Resolved in 0289e8495254ef2b2a1f33c567422bc1fade9a57.